docs/content/preview/drivers-orms/ruby/activerecord.md

```diff
@@ -96,8 +128,14 @@ Type "help" for help.
 yugabyte=#
 ```
 
-```plpgsql
-yugabyte=# SELECT count(*) FROM users;
+Connect to the database that is mentioned in `config/database.yml` file. Default is `ysql_active_record`.
```

Comment:
there is a file named in database.yml? why can't we say what it is? is it the same as the default, or does the default refer to the file in database.yml? if so, who would have changed it?

Reply:
We mention about what it is on line 30 and that it can be customized by the user.